src/seahawk/seahawk_deck/thrust.py

```diff
@@ -39,28 +44,223 @@ class Thrust(Node):
     def __init__(self):
         """Initialize this node"""
         super().__init__('thrust')
+
+        self.MAX_FWD_THRUST = 36.3826715 # N
+        self.MAX_REV_THRUST = -28.6354180 # N
+
+        self.TOTAL_CURRENT_LIMIT = 70 # A
+        self.ESC_CURRENT_LIMIT = 40 # A
+
+        self.motor_positions = [
+            [0.19, 0.12, 0.047],
+            [0.19, -0.12, 0.047],
+            [-0.19, 0.12, 0.047],
+            [-0.19, -0.12, 0.047],
+            [0.105, 0.15, -0.038],
+            [0.105, -0.15, -0.038],
+            [-0.105, 0.15, -0.038],
+            [-0.105, -0.15, -0.038]
+        ]
+
+        self.motor_thrusts = [
+            [0.0, 0.0, 1.0],
+            [0.0, 0.0, 1.0],
+            [0.0, 0.0, 1.0],
+            [0.0, 0.0, 1.0],
+            [0.7071, -0.7071, 0],
+            [0.7071, 0.7071, 0],
+            [-0.7071, -0.7071, 0],
+            [-0.7071, 0.7071, 0]
+        ]
+
+        self.declare_parameter('center_of_mass_offset', [0.0, 0.0, 0.0])
+        self.center_of_mass_offset = self.get_parameter('center_of_mass_offset').value
+
+        self.add_on_set_parameters_callback(self.update_center_of_mass)
+
+        self.motor_config = self.generate_motor_config()
+        self.inverse_config = np.linalg.pinv(self.motor_config, rcond=1e-15, hermitian=False)
+
         self.subscription = self.create_subscription(Twist, 'drive/twist', self._callback, 10)
         self.motor_pub = self.create_publisher(Float32MultiArray, 'drive/motors', 10)
```

Comment:
Consider making the topic name more descriptive. And change string to use double quotes for consistency